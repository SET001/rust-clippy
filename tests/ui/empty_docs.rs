#![allow(unused)]
#![warn(clippy::empty_docs)]

pub mod outer_module {

    //!

    //! valid doc comment

    //!!

    //!! valid doc comment

    ///

    /// valid doc comment

    /**
     *
     */

    /**
     * valid block doc comment
     */

    pub mod inner_module {}
}
