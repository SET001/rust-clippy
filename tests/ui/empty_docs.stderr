error: empty doc comment
  --> $DIR/empty_docs.rs:14:5
   |
LL |     ///
   |     ^^^
   |
   = help: consider removing it or fill it in
   = note: `-D clippy::empty-docs` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::empty_docs)]`

error: empty doc comment
  --> $DIR/empty_docs.rs:18:5
   |
LL | /     /**
LL | |      *
LL | |      */
   | |_______^
   |
   = help: consider removing it or fill it in

error: empty doc comment
  --> $DIR/empty_docs.rs:6:5
   |
LL |     //!
   |     ^^^
   |
   = help: consider removing it or fill it in

error: empty doc comment
  --> $DIR/empty_docs.rs:10:5
   |
LL |     //!!
   |     ^^^^
   |
   = help: consider removing it or fill it in

error: aborting due to 4 previous errors

